### PR TITLE
Random mappings

### PR DESCRIPTION
**Pull request by anonymous123-code**
- World properties
- Explosions
- Projectiles
- Breeze
- wind charge
- world properties

### CODE REVIEW

File: mappings/net/minecraft/entity/Entity.mapping [hunk 1]
```diff
@@ -659,6 +659,8 @@ CLASS net/minecraft/unmapped/C_astfners net/minecraft/entity/Entity
 	METHOD m_plmkvbcv streamIntoPassengers ()Ljava/util/stream/Stream;
 	METHOD m_pnapxqoc updatePassengerPosition (Lnet/minecraft/unmapped/C_astfners;)V
 		ARG 1 passenger
+	METHOD m_psgotipc getDeflectionBehavior (Lnet/minecraft/unmapped/C_ltpsyvhj;)Lnet/minecraft/unmapped/C_jslwmoew;
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/entity/Entity.mapping:662]: #553 calls this `getProjectileDeflector`, I can't check source rn to see which I prefer

File: mappings/net/minecraft/entity/projectile/DeflectionBehavior.mapping [hunk 2]
```diff
@@ -0,0 +1,9 @@
+CLASS net/minecraft/unmapped/C_jslwmoew net/minecraft/entity/projectile/DeflectionBehavior
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/entity/projectile/DeflectionBehavior.mapping:1]: #553 calls this `ProjectileDeflector`

File: mappings/net/minecraft/world/explosion/Explosion.mapping [hunk 3]
```diff
@@ -33,22 +46,45 @@ CLASS net/minecraft/unmapped/C_aahhrzpf net/minecraft/world/explosion/Explosion
...
 	METHOD m_hyeznwfn getExposure (Lnet/minecraft/unmapped/C_vgpupfxx;Lnet/minecraft/unmapped/C_astfners;)F
 		ARG 0 source
 		ARG 1 entity
 	METHOD m_jgqsaodd getPosition ()Lnet/minecraft/unmapped/C_vgpupfxx;
+	METHOD m_mxankppo getDamageSource (Lnet/minecraft/unmapped/C_cdctfzbn;Lnet/minecraft/unmapped/C_astfners;)Lnet/minecraft/unmapped/C_sbxfkpyv;
+		ARG 1 source
 	METHOD m_nbqwldyc getPower ()F
 	METHOD m_oiheoshm getCausingEntity ()Lnet/minecraft/unmapped/C_usxaxydn;
 	METHOD m_oivdgoim collectBlocksAndDamageEntities ()V
+	METHOD m_opijfldo getCausingEntity (Lnet/minecraft/unmapped/C_astfners;)Lnet/minecraft/unmapped/C_usxaxydn;
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/world/explosion/Explosion.mapping:75]: `CausingEntity` is a little awkward, would `Owner` 0r `Igniter` make sense?

**Review by supersaiyansubtlety — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/entity/projectile/DeflectionBehavior.mapping (hunk 2 above)
Comment (supersaiyansubtlety) [reply mappings/net/minecraft/entity/projectile/DeflectionBehavior.mapping:1]: Given it has a `deflect` method, `ProjectileDeflector` seems more appropriate (but can't check source rn)

**Comment by anonymous123-code** [comment]
This PR duplicates some stuff done in #553, Ill wait until that one gets merged before updating this

**Comment by eristhea** [comment]
@anonymous123-code #553 has been merged, feel free to finish up this PR!

**Comment by eristhea** [comment]
mostly superceded by various PRs

**Pull request closed without merging**